.babelrc

```diff
@@ -0,0 +1,5 @@
+{

```

Comment:
Why revert back to `.babelrc`?

Reply:
3 Reasons

1. It is the [standard for babel when not wanting to compile node_modules (which we don't)](https://babeljs.io/docs/en/configuration)
2. Support for plugins / build in features that IDE's such as VSCode and IntelliJ IDEA have to recognise it as a babel configuration and provide proper IntelliSense
3. Support for plugins such as Material Icons in common IDE's to recognise it as a Babel file. Visual only but nice regardless.

Reply:
Ok sounds good.